plugin/storage/es/mappings/mapping.go

```diff
@@ -46,10 +46,8 @@ type MappingBuilder struct {
 func (mb *MappingBuilder) GetMapping(mapping string) (string, error) {
 	if mb.EsVersion == 8 {
 		return mb.fixMapping(mapping + "-8.json")
-	} else if mb.EsVersion == 7 {
-		return mb.fixMapping(mapping + "-7.json")
 	}
-	return mb.fixMapping(mapping + ".json")
```

Comment:
the json mapping files should be deleted as well

Reply:
Yes, I will remove them as well.